### DISCUSSION TITLE
going to use this on a daily basis

### DISCUSSION THREAD

**Discussion by RjGunner111** [justcoding121/Titanium-Web-Proxy#953] [Q&A] [answered]
is this safe to use on a daily basis? because I am planning to use this to prevent myself from going to undisclosed websites and I want to be sure that I won't be hit with anything should I develop this into a daily tool for me to use

**Comment by justcoding121** [comment, marked as answer]
Yeah, plenty of people run it daily for personal tooling ? the library itself is fine for that.

A few practical notes though:
- Blocking sites is on you (check host/URL in `BeforeRequest` / `BeforeTunnelConnectRequest` and return a block response). The proxy doesn't come with a built-in "safe browsing" filter.
- For HTTPS inspection you'll install/trust a root cert ? only do that on machines you control, and understand what MITM means.
- Keep the package updated, and don't expose the proxy to the public internet without auth.

If you're just filtering your own browser traffic locally, you're in a pretty normal use case. Closing this out ? good luck with the tool!
